alchemist/src/orchid/resources/wiki/summary.md

```diff
@@ -4,7 +4,7 @@
   - [The metamodel](simulator/metamodel.md)
 - [Using Alchemist](usage.md)
   - [Requirements and installation](usage/installation.md)
-  - [Quick start](usage/quickstart.md)
+  - [How to run simulations](usage/run.md)
```

Comment:
Uhmm... why? I like the idea of having a quick start

Reply:
I agree, the quick start section was added back and filled with a minimal use-case explained